components/01-atoms/links/_link.scss

```diff
@@ -0,0 +1,59 @@
+@mixin animation-link($property) {
+  transition-property: #{$property};
+  transition-duration: 0.2s;
+  transition-timing-function: ease-in-out;
+}
+
+a {
```

Comment:
@callinmullaney I'm styling the `<a>` tag directly for now. I expect I'll have to refactor it and scope it to `.text-field a` or something like that eventually, but until I hit an issue and/or create the general `text` component that would implement that, I don't know how that should be handled yet.